### ISSUE TITLE
Not GND related subject chain keyword types (ALMA)

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1202]
Was part of #1176

We have subject components that do not relate to GND but that are given a type character in ALMA/MARC.
These have type specifications in subfield A and not in subfield D.

e.g.:
```XML
  <datafield tag="689" ind1="0" ind2="2">
    <subfield code="a">Online-Ressource</subfield>
    <subfield code="A">f</subfield>
  </datafield>
  <datafield tag="689" ind1="0" ind2="3">
    <subfield code="a">Zeitschrift</subfield>
    <subfield code="A">f</subfield>
  </datafield>
```

Subfield "A" refers to values-list:
f - Formschlagwort
g - geographischer Unterbegriff (Veranstaltungsort  und Veranstaltungsjahr)
z - Zeitschlagwort

Our type-vocabulary (almost identical to the vocab list in Subfield D) allows for 
https://github.com/hbz/lobid-resources/blob/352d5f628046f05eb7b191202163bcd29dc13708/src/test/resources/schemas/complexSubject.json#L35-L43

@acka47 How should I type Formatschlagwort, Zeitschlagwort and Veranstaltungsort? Should I only add them to the enum-List? If so, do you have suggestions what kind of type I should use or where I could look for possible concepts?

g could be sorted to "PlaceOrGeographicName" as suggested here: https://www.gbv.de/bibliotheken/verbundbibliotheken/02Verbund/01Erschliessung/02Richtlinien/01KatRicht/559x.pdf (P.2)

But temporal and formal characteristic would still be missing.

_Originally posted by @TobiasNx in https://github.com/hbz/lobid-resources/issues/1176#issuecomment-817560872_

**Comment by acka47** [comment]
As far as I know, these Formschlagwörter and Geographica should have a correspoding entry in the GND:

- Online-Ressource:  https://lobid.org/gnd/4511937-5
- Zeitschrift: https://lobid.org/gnd/4067488-5

I assume that all "Formschlagwörter" are typed as `SubjectheadingSensoStricto`, a GND search confirms that: https://lobid.org/gnd/search?q=formschlagwort&size=100&format=html. Aren't the GND-IDs for these included in the Alma MARC? If not, we could create a mapping.

Re. Geographica, I am not sure whether this is acutally used very often. It doesn't look like it as I can't find an example in lobid. We should not assign a type if we only have the string and no ID.

Re. Zeitschlagwort,  think these contain a year range like "1870-1900" or "19 Jhd.". Here is an example how we currently deal with it: https://lobid.org/resources/HT002288207

**Comment by TobiasNx** [comment]
This ticket can be closed via #2031 since it resolves it like in the old morph giving all non GND subjects the standard type `SubjectHeading`
